docs/API.md

```diff
@@ -107,17 +107,23 @@ A `<Link>` can know when the route it links to is active and automatically apply
 
 #### Props
 ##### `to`
-The path to link to, e.g. `/users/123`.
-
-##### `query`
+Can be either a string or an object.
```

Comment:
I actually like what you currently have. I'd just add a caveat saying that this value is really just a location descriptor, and in the case of a custom history can be an arbitrary whatever.